test/k8sT/manifests/echo-hostnetns-svc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: echo-hostnetns
+spec:
+  selector:
+    matchLabels:
+      name: echo-hostnetns
+  template:
+    metadata:
+      labels:
+        name: echo-hostnetns
+    spec:
+      hostNetwork: true
+      containers:
+      - name: echo-container
+        image: docker.io/cilium/echoserver:1.10.2
+        env:
+        - name: PORT
+          value: "6666"
+        livenessProbe:
+          exec:
+            command:
+            - curl
+            - -sS
+            - -o
+            - /dev/null
+            - localhost:6666
+        readinessProbe:
+          exec:
+            command:
+            - curl
+            - -sS
+            - -o
+            - /dev/null
+            - localhost:6666
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo-hostnetns
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 6666
```

Comment:
Shouldn't this be `6667`? How are we possibly testing this if we are opening a NodePort on the same port as a pod running in the host network is listening for requests?

Reply:
This is for the `ClusterIP` svc. Also, because `targetPort` is not specified, it will be derived from `port`.